resources/lib/videoplayer.py

```diff
@@ -59,6 +60,7 @@ def start_playback(self):
 
         self._handle_resume()
         self._handle_skipping()
+        self._handle_upnext()
```

Comment:
I'm thinking of making this configurable via settings, but not sure if that is something people would like to configure.

Reply:
Now configurable and advised in VideoStream._get_upnext_episode.